test/functional/mining_prioritisetransaction.py

```diff
@@ -263,6 +263,12 @@ def run_test(self):
             if (x != high_fee_tx):
                 assert x not in mempool
 
+
+        self.log.info("Assert that prioritised free transaction with 0 delta is never added to mapDeltas")
+        tx_id_zero_del = self.wallet.create_self_transfer(fee_rate=0)['txid']
```

Comment:
in a1fb3eddc91db708727fc48309d3bb6182858bbd

I don't think it matters what the feerate of this tx is?
```suggestion
        self.log.info("Assert that 0 delta is never added to mapDeltas")
        tx_id_zero_del = self.wallet.create_self_transfer()['txid']
```

Reply:
yes you are right updated the test in [0d3a4ab](https://github.com/bitcoin/bitcoin/pull/28885/commits/0d3a4abd805e3d4deec25f9902e762a7e693ce6b)